website/docs/tutorials/06-deploying/02-migrations/declare.md

```diff
@@ -2,17 +2,19 @@
 
 ```python
 def declare(
-    contract_path: str,
+    contract_path_str: str,
     *,
     config: Optional[CheatcodeNetworkConfig] = None
 ) -> DeclaredContract:
 
 class DeclaredContract:
     class_hash: int
 ```
-Declares contract given a relative to a project root path to **compiled** contract.
 
-`config` is a keyword only argument that allows passing [network configuration](../03-network-config.md) data. See related documentation for more information.
+Send a declare transaction.
```

Comment:
```suggestion
Sends a declare transaction.
```

Reply:
- https://starknetpy.readthedocs.io/en/latest/gateway_client.html?highlight=declare#starknet_py.net.gateway_client.GatewayClient.declare
- We should also change it for the deploy-contract cheatcode to be consistent.